x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -14393,7 +14393,6 @@
     "xpack.lens.metric.addLayer": "ビジュアライゼーションレイヤーを追加",
     "xpack.lens.metric.groupLabel": "表形式の値と単一の値",
     "xpack.lens.metric.label": "メトリック",
-    "xpack.lens.pageTitle": "レンズ",
```

Comment:
No need to delete these translations now that we reverted the tab title change :)

Reply:
You are right!